components/tag/style/index.less

```diff
@@ -29,14 +29,6 @@
     color: @tag-default-color;
   }
 
-  &-text {
-    a:first-child:last-child {
-      display: inline-block;
-      margin: 0 -8px;
-      padding: 0 8px;
-    }
-  }
-
```

Comment:
@afc163 it's seems unnecessary ?  https://ant.design/components/tag-cn/#components-tag-demo-basic  remove this style, work fine

Reply:
It is useful for `<Tag><a href="">link</a></Tag>`:

<img width="110" alt="image" src="https://user-images.githubusercontent.com/507615/35334341-0113e4ec-014d-11e8-8647-7ed0809a19a5.png">

The hack code will fill the whole area inside Tag, otherwise it could be unclickable at the edge.

Reply:
Yeah, I inspected these styles before PR. The .ant-tag-text is necessary.
BTW, @afc163 is this RP in progress?

Reply:
@zheeeng  take a look for `<Tag><a href="">link</a></Tag>`, the `a` must be fill the Tag block.